terraform/app/modules/paas/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = "~> 0.14.5"
+  required_providers {
```

Comment:
Comment really, but will this version number need to be updated
 over the years ?

Reply:
This is more to lock the version to not introduce breaking changes.

If it's required to run on another version of Terraform, this should be updated